frontend/src/features/workflow-viewer/ui/GraphViewer.module.css

```
@@ -3,43 +3,56 @@
   height: 100%;
   min-height: 500px;
   position: relative;
-  background: var(--bg-color);
+  background: var(--node-canvas-bg);
   border: 1px solid var(--glass-border);
-  border-radius: var(--radius-md);
+  border-radius: var(--radius-card);
   overflow: hidden;
 }
 
 .container :global(.react-flow__edge-path) {
```

Stylelint (17.11.1)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)